indico/modules/rb_new/client/js/common/map/selectors.js

```diff
@@ -44,3 +44,5 @@ export const makeGetMapData = (namespace) => ({[namespace]: stateSlice}) => ({
 });
 
 export const makeIsMapSearchEnabled = (namespace) => ({[namespace]: stateSlice}) => stateSlice.map.search;
+
+export const getHoveredRoom = ({map: {ui}}) => ui.hoveredRoom;
```

Comment:
I would go for this:

```js
export const getHoveredRoom = ({map}) => map.ui.hoveredRoom;
```

Like this it's consistent with all the other selectors and it's easy to find accesses to a given state slice by searching for "foo.bar" in the code